build.gradle.kts

```diff
@@ -34,6 +39,18 @@ allprojects {
     }
 }
 
+nexusPublishing {
+    repositories {
+        sonatype {
+            val sonatypeUsername = System.getenv("OSSRH_USERNAME")
+            val sonatypePassword = System.getenv("OSSRH_PASSWORD")
+            stagingProfileId.set("378eecbbe2cf9")
```

Comment:
what is the significance of this hardcoded `id` ?

Reply:
It's the ID we can use to send data to our repository